SharedCode/src/commonMain/kotlin/ApplicationPresenter.kt

```diff
@@ -8,12 +8,34 @@ class ApplicationPresenter: ApplicationContract.Presenter() {
     private val dispatchers = AppDispatchersImpl()
     private var view: ApplicationContract.View? = null
     private val job: Job = SupervisorJob()
+    private val stations = listOf(
+        "BON",
+        "KGX",
+        "EUS",
+        "MAN",
+        "EDB"
+    )
+
+    private val stationSubmitButtonText = "Search"
```

Comment:
Do we ever change this? If not it might be nice to just have this hardcoded in the storyboard/xml files

Reply:
We think it's helpful to have it here for in future when we want to make it say 'loading' during the api request.

Reply:
Nice! (You could look into some interesting ways to show loading state, but this sounds like a nice approach)